appengine/go11x/warmup/app.yaml

```diff
@@ -0,0 +1,12 @@
+runtime: go111
+
+inbound_services:
+  - warmup
+
+automatic_scaling:
+  min_idle_instances: 1
```

Comment:
Why is this needed?

Reply:
I think it is needed, since warmup requests are for priming idle instances.

It'd be good to add a comment or link to the docs.

Reply:
I've incorporated some feedback, but am holding off on linking to docs while I work out what docs should be linked.

Also working to confirm this is how we want to use warmup in go.